### ISSUE TITLE
`fab serve` errors can be unclear

### ISSUE THREAD

**Issue by iand675** [fab-spec/fab#256]
I'm trying to get a next.js app running, but when I run `fab serve fab.zip`, it gives a relatively unclear error:

```
yarn run v1.22.4
$ /Users/ian/Code/front-door/node_modules/.bin/fab serve ./fab.zip
        [Server] 💎 fab serve 💎
        [Server] Reading ./fab.zip…
        [Server] ✔ Done. Booting FAB server…
    TypeError: Object prototype may only be an Object or null: undefined
error Command failed with exit code 1.
```

Ideally I'd like to see here: 
- stack traces
- whether this error is coming from my code or `fab serve` or the `fab` runtime code
- (nice to have) ability to drop into a node repl or something to do a bit of hands-on debugging

**Comment by evanderkoogh** [comment]
Ohh.. I *really* like the reply idea. An easy way to throw Requests at the fab programmatically.. without having to mess around with curl and host headers etc.. 

But first order of business would indeed be to give you a bit more clarity and certainty were the errors are.

**Comment by Rashmi-278** [comment]
I encountered the same issue , 
@iand675  can you please your solution for this if you happened to solve it ?

**Comment by evanderkoogh** [comment]
Hey @iand675 and @Rashmi-278. Apologies, I thought my PR #260 was already merged, but it had some issues. I will fix those today and ask @geelen to do a patch release for this.

**Comment by geelen** [comment]
Ok #260 still has some problems but I've solved problems like this by jumping into your `node_modules/@fab/server` directory, then opening `lib/index.js` and throwing some `console.log` statements around. You should be able to track down the line it's failing on so we can solve your particular issue, then we can solve the debugging experience in #260.

Also, there's a very small chance that `esm/index.js` is the file that's being loaded: every package in FAB is compiled for both Node's require and ESM modules but I'm 99% the Node one is being used (the `lib` dir) but 99 isn't 100 so...

**Comment by geelen** [comment]
Ok #260 is merged and `@fab/server@1.0.0-rc.7.p1` released. Give it a try and see if the error messages are any better?
